x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/planner/QueryTranslator.java

```diff
@@ -144,7 +150,7 @@
     }
 
     static QueryTranslation toQuery(Expression e, boolean onAggs) {
-        QueryTranslation translation = null;
+        QueryTranslation translation;
```

Comment:
Looks like a formatting issue - I'd keep the explicit initialization (to keep introspection tools at bay that yes, `null` is expected).